serialization/src/main/scala/sbt/serialization/SerializedValue.scala

```diff
@@ -145,6 +153,17 @@ private final case class LazyValue[V](value: V, pickler: Pickler[V]) extends Ser
     unpicklerMatchesExactly(unpickler) || toJson.hasTag[T]
 
   override def toJsonString = toJson.toJsonString
+  override def toJsonFile(file: File): Unit =
+    {
+      val output = new UFT8FileOutput(file)
+      try {
+        val builder = json.pickleFormat.createBuilder(output)
```

Comment:
Nice.  So this should save a good bit of memory by dumping stright to the OutputStream rather than putting into a StringBuffer, then dumping a giant string.